waspc/README.md

```diff
@@ -17,150 +17,135 @@ If you would like to make your first contribution, here is a handy checklist we
 
 ## Quick overview
 
-Wasp compiler is implemented in Haskell, but you will also see a lot of Javascript and other web technologies because Wasp compiles it's own code into them.
+The core of `waspc` is implemented in Haskell, but you will also see a lot of Javascript/TypeScript and other web technologies since Wasp generates web apps in those.
 
-You don't have to be expert in Haskell to contribute or understand the code, since we don't use complicated Haskell features much -> most of the code is relatively simple and straight-forward, and we are happy to help with the part that is not.
+You don't have to be expert in Haskell to contribute or understand the code, since we don't use complicated Haskell features much -> most of the code is relatively simple and straight-forward, and we are happy to help with any parts that are not.
 
-Main result of building the project is `wasp` executable (also referred to as CLI), which is both Wasp compiler, CLI and Wasp project runner in one - one tool for everything Wasp-related.
+Main result of building the project is `wasp` executable (also referred to as CLI), which is both Wasp compiler, CLI and Wasp project runner in one - one main tool for everything Wasp-related.
 
-`wasp` executable takes `.wasp` files and `ext/` dir as input and generates a web app from them.
-
-![Wasp compilation](images/wasp-diagram.png)
-
-It can then also run that web app for you, deploy it (not yet but that is coming), and manage it in other ways.
+Note that while `waspc` is the name of this Haskell project, the executable's name is `wasp` when released, and `wasp-cli` during development.
 
 ## Basics
 
-### Setup
-
-We use the [`cabal`](https://cabal.readthedocs.io/) command line tool to build
-the project. The best way to install it is via
-[ghcup](https://www.haskell.org/ghcup/). The `ghcup tui` terminal user interface
-is a convenient way of installing and selecting versions of `cabal`, `hls` and
-`ghc`.
-
-:warning: On Mac, we recommend using the official [ghcup](https://www.haskell.org/ghcup/) installer over Homebrew, as it works out of the box.
-
-Make sure to use `cabal` CLI version >=3.12, since we use some of the newer cabal functionalities.
-
-In [cabal.project](cabal.project) we're explicitly saying what compiler version
-to build Wasp with.
+### Repo
 
-```cabal
-with-compiler: ghc-x.y.z
-```
+Clone this repo (or its fork) to your machine.
 
-Please ensure that you have this ghc version installed.
+Position yourself in this directory (`waspc/`) and make sure that you are on the `main` branch.
 
-### Repo
+### `run` script
```

Comment:
No we introduce `run` script much earlier, at the very start. Which makes sense, since the rest of this file will be mentioning it a lot.